ipaserver/install/plugins/update_ra_cert_store.py

```diff
@@ -2,74 +2,61 @@
 # Copyright (C) 2016  FreeIPA Contributors see COPYING for license
 #
 
-import binascii
 import os
+import tempfile
 
 from ipalib import Registry
 from ipalib import Updater
-from ipalib.constants import IPAAPI_USER, IPAAPI_GROUP
 from ipalib.install import certmonger
 from ipaplatform.paths import paths
-from ipapython import certdb
+from ipapython.certdb import NSSDatabase
+from ipaserver.install import cainstance
 
 register = Registry()
 
 
 @register()
 class update_ra_cert_store(Updater):
     """
-    Moves the cert store from /etc/httpd/alias to /var/lib/ipa/radb
+    Moves the ipaCert store from /etc/httpd/alias RA_AGENT_PEM, RA_AGENT_KEY
+    files
     """
 
     def execute(self, **options):
+        ra_nick = 'ipaCert'
         ca_enabled = self.api.Command.ca_is_enabled()['result']
         if not ca_enabled:
             return False, []
 
-        olddb = certdb.NSSDatabase(nssdir=paths.HTTPD_ALIAS_DIR)
-        if not olddb.has_nickname('ipaCert'):
+        certdb = NSSDatabase(nssdir=paths.HTTPD_ALIAS_DIR)
+        if not certdb.has_nickname(ra_nick):
             # Nothign to do
             return False, []
 
-        newdb = certdb.NSSDatabase(nssdir=paths.IPA_RADB_DIR)
-        if os.path.exists(paths.IPA_RADB_DIR):
-            if newdb.has_nickname('ipaCert'):
+        if os.path.exists(paths.RA_AGENT_PEM):
+            if certdb.has_nickname(ra_nick):
```

Comment:
This if statement is superfluous - the condition will always be true, because it is verified to be true above.